src/terraform.js

```diff
@@ -36,8 +36,27 @@ const init = async (opts = {}) => {
 };
 
 const apply = async (opts = {}) => {
-  const { dir = './' } = opts;
-  return await exec(`terraform -chdir='${dir}' apply -auto-approve`);
+  const { dir = './', json = false } = opts;
+  if (json) {
+    const env = process.env;
+    if (env.TF_LOG_PROVIDER === undefined) env.TF_LOG_PROVIDER = 'DEBUG';
+    try {
+      await tfCapture(
+        'terraform',
+        [`-chdir='${dir}'`, 'apply', '-auto-approve', '-json'],
+        {
+          cwd: process.cwd(),
+          env: env,
```

Comment:
`-Wrepetitive-although-consistent`
```suggestion
          env,
```